private/bufpkg/bufmodule/bufmodule.go

```diff
@@ -110,6 +111,10 @@ type Module interface {
 	//
 	// This may be nil, since older versions of the module would not have this stored.
 	LintConfig() *buflintconfig.Config
+	// Manifest returns the manifest for the module (possibly empty).
+	Manifest() manifest.Manifest
+	// Blobs return the raw data for the module (possibly empty).
+	Blobs() manifest.BlobSet
```

Comment:
I guess we'll need to have some backend PR as well to cover this change, ready to be merged when this one is?

Reply:
Will we rely on having these available in the backend for anything? We're only using them in the caching logic right now.